ChangeFeedLabSolution/local.settings.json

```diff
@@ -7,8 +7,6 @@
     // Connection string from Azure Storage account
     "AzureWebJobsDashboard": "your Storage connection string here",
     // Primary connection string from Azure Cosmos DB account
-    "DBConnection": "your Cosmos DB connection string here",
-    // Connection string-primary key from Azure Event Hub namespace
-    "EventHubNamespaceConnection": "your Event Hub Namespace connection string here"
+    "DBConnection": "your Cosmos DB connection string here"

```

Comment:
By [spec](https://www.json.org/json-en.html), JSON does not support embedded comments, even though some parsers tolerate them.   We should stick to standard JSON and remove the comments.

_([additional context](https://web.archive.org/web/20150105080225/https://plus.google.com/+DouglasCrockfordEsq/posts/RK8qyGVaGSr))_

Reply:
@jsquire - Removed.